src/main/java/dasniko/keycloak/authenticator/SmsAuthenticator.java

```diff
@@ -89,7 +94,8 @@ public boolean configuredFor(KeycloakSession session, RealmModel realm, UserMode
 	}
 
 	private boolean isValidPhoneNumber(String phoneNumber) {
-			Matcher validPhoneNumber =  PHONE_NUMBER_FORMAT.matcher(phoneNumber);
-			return validPhoneNumber.find();
+		Matcher validPhoneNumber = REGEX_PHONE_NUMBER.matcher(phoneNumber);
+		return validPhoneNumber.find();
```

Comment:
should also include an `if`  to check that `phoneNumber` is not null,
To make sure the full string matches I'd suggest to use `matches()` method instead of `find()`